Src/SDK/Common/Temporal.Common/public/Payload.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Candidly.Util;
+
+using Temporal.Common.Payloads;
+
+namespace Temporal.Common
+{
+    public static partial class Payload
+    {
+        public static readonly IPayload.Void Void = IPayload.Void.Instance;
+        public static readonly Task<IPayload.Void> VoidTask = IPayload.Void.CompletedTask;
+
+        #region Unnamed(..)
+
+        public static object Named<T>(params object[] namedBalues)
```

Comment:
Assuming you mean this:

```suggestion
        public static object Named<T>(params object[] namedValues)
```

Reply:
✔️